### PR TITLE
feat(vm): provided sysprep ability

### PR DESCRIPTION
**Pull request by Isteb4k**
## Description
Added sysprep ability for windows installation

## Why do we need it, and what problem does it solve?
To provide Sysprep ability it's necessary to define in virtual machine with SysprepSecret provisioning. This PR adds SysprepSecret implementation

## Checklist
- [ ] The code is covered by unit tests.
- [X] e2e tests passed.
- [X] Documentation updated according to the changes.
- [X] Changes were tested in the Kubernetes cluster manually.

### CODE REVIEW

File: docs/FAQ_RU.md [hunk 1]
```diff
@@ -224,3 +224,56 @@ kubectl get pvc dvcr -n d8-virtualization
 NAME   STATUS   VOLUME                                     CAPACITY     ACCESS MODES   STORAGECLASS            AGE
 dvcr   Bound    pvc-6a6cedb8-1292-4440-b789-5cc9d15bbc6b   57617188Ki   RWO            linstor-thick-data-r1   7d
 ```
+
+## Как предоставить файл ответов windows (Sysprep)
+
+Чтобы предоставить виртуальной машине windows файл ответов необходимо указать provisioning с типом SysprepSecret.
+
+Предже всего необходимо создать секрет::
```
Comment (universal-itengineer) [inline docs/FAQ_RU.md:232]:
```suggestion
Предже всего необходимо создать секрет:
```

File: images/virtualization-artifact/pkg/controller/kvbuilder/kvvm_compare.go [hunk 2]
```diff
@@ -205,6 +205,14 @@ func CompareKVVM(curr, next *KVVM) (*ChangeApplyActions, error) {
 		actions.Add(action)
 	}
 
+	{
+		action, err := CompareSysprep(curr, next)
```
Comment (yaroslavborbat) [inline images/virtualization-artifact/pkg/controller/kvbuilder/kvvm_compare.go:209]: looks like Legacy. Now we don't use CompareKvvm. You should add sysprep to pkg vmchange.

**Review by yaroslavborbat — CHANGES_REQUESTED** [review]

File: docs/FAQ_RU.md [hunk 3]
```diff
@@ -224,3 +224,56 @@ kubectl get pvc dvcr -n d8-virtualization
 NAME   STATUS   VOLUME                                     CAPACITY     ACCESS MODES   STORAGECLASS            AGE
 dvcr   Bound    pvc-6a6cedb8-1292-4440-b789-5cc9d15bbc6b   57617188Ki   RWO            linstor-thick-data-r1   7d
 ```
+
+## Как предоставить файл ответов windows (Sysprep)
+
+Чтобы предоставить виртуальной машине windows файл ответов необходимо указать provisioning с типом SysprepSecret.
+
+Прежде всего необходимо создать секрет::
```
Comment (universal-itengineer) [inline docs/FAQ_RU.md:232]:
```suggestion
Прежде всего необходимо создать секрет:
```

**Review by yaroslavborbat — APPROVED** [review]

**Review by universal-itengineer — APPROVED** [review]

File: docs/FAQ.md [hunk 4]
```diff
@@ -182,3 +182,56 @@ metadata:
...
+kind: Secret
+metadata:
+  name: sysprep-config
+data:
+  unattend.xml: XXXx # base64 of answer file 
+```
+
+And then feel free to create a virtual machine with unattended installation.
+
+It's necessary to make sure the virtual machine does not restart, which can be done by setting the run policy as AlwaysOn:
```
Comment (diafour) [inline docs/FAQ.md:203]: Is it about restart? "AlwaysOn" doesn't prevent restarting from inside the VM.

File: docs/FAQ.md [hunk 5]
```diff
@@ -182,3 +182,56 @@ metadata:
...
     app: old
 ```
+
+## How to provide windows answer file (Sysprep)
+
+To provide Sysprep ability it's necessary to define in virtual machine with SysprepSecret provisioning.
+Set answer files (typically named unattend.xml or autounattend.xml) to secret to perform unattended installations of Windows.
+You can also specify here other files in base64 format (customize.ps1, id_rsa.pub, ...) that you need to successfully execute scripts inside the answer file.
+
+First of all create sysprep secret:
```
Comment (diafour) [inline docs/FAQ.md:192]:
```suggestion
First, create sysprep secret:
```

File: docs/FAQ.md [hunk 6]
```diff
@@ -182,3 +182,56 @@ metadata:
...
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: sysprep-config
+data:
+  unattend.xml: XXXx # base64 of answer file 
+```
+
+And then feel free to create a virtual machine with unattended installation:
```
Comment (diafour) [inline docs/FAQ.md:203]:
```suggestion
Then create a virtual machine with unattended installation:
```

File: docs/FAQ_RU.md [hunk 7]
```diff
@@ -224,3 +224,56 @@ kubectl get pvc dvcr -n d8-virtualization
...
+metadata:
+  name: sysprep-config
+data:
+  unattend.xml: XXXx # base64 файла ответов
+```
+
+Затем можно создать виртуальную машину, которая в процессе установке будет использовать файл ответов.
+
+Необходимо убедиться, чтобы виртальная машина не перезапускалась, что возможно обеспечить установив runPolicy в значение AlwaysOn: 
+
```
Comment (diafour) [inline docs/FAQ_RU.md:246]:
```suggestion
```

**Review by diafour — APPROVED** [review]

**Pull request merged**